### PR TITLE
Fixed class naming in Relationships Unmanaged Unit Test

### PR DESCRIPTION
**Pull request by MatthewBlanski**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

**Review by xinnanlisf — APPROVED** [review]
Looks like this class is prettiered. Not sure if we want to but since it is unpackaged, oh well

File: unpackaged/test_packaged/relationship_autocreate/classes/REL_Relationships_Con_PTST.cls [hunk 1]
```diff
@@ -28,38 +28,37 @@
...
-    *********************************************************************************************************/
+     * @description Unpackaged test to verify that when Relationship Auto Creation is enabled, and multiple
+     * mappings exist, inserting a Contact record with all Auto Creation mapped fields populated generates
+     * relationship  and reciprocal relationship records for each mapped field.
+     *********************************************************************************************************/
     @isTest
-    private static void insertContactWithMultipleAutoCreateMappingsAllPopulated(){
-        REL_Relationships_Con_AfterInsert_MTST.setupMultipleAutoCreateRelationships();
+    private static void insertContactWithMultipleAutoCreateMappingsAllPopulated() {
+        REL_Relationships_Con_PTST.setupMultipleAutoCreateRelationships();
```
Comment (MatthewBlanski) [inline unpackaged/test_packaged/relationship_autocreate/classes/REL_Relationships_Con_PTST.cls:48]: This is the first actual code change so the code references the appropriate class

File: unpackaged/test_packaged/relationship_autocreate/classes/REL_Relationships_Con_PTST.cls [hunk 2]
```diff
@@ -70,71 +69,94 @@ private class REL_Relationships_Con_PTST {
...
-    * relationship and reciprocal relationship records only for the populated mapped fields.
-    *********************************************************************************************************/
+     * @description Unpackaged test to verify that when Relationship Auto Creation is enabled, and multiple
+     * mappings exist, inserting a Contact record some Auto Creation mapped fields populated generates
+     * relationship and reciprocal relationship records only for the populated mapped fields.
+     *********************************************************************************************************/
     @isTest
-    private static void insertContactWithMultipleAutoCreateMappingsSomePopulated(){
-
-        REL_Relationships_Con_AfterInsert_MTST.setupMultipleAutoCreateRelationships();
```
Comment (MatthewBlanski) [inline unpackaged/test_packaged/relationship_autocreate/classes/REL_Relationships_Con_PTST.cls:129]: The second actual code change

File: unpackaged/test_packaged/relationship_autocreate/classes/REL_Relationships_Con_PTST.cls [hunk 3]
```diff
@@ -144,57 +166,71 @@ private class REL_Relationships_Con_PTST {
...
-    * generate any relationships records.
-    *********************************************************************************************************/
+     * @description Unpackaged test to verify that when Relationship Auto Creation is enabled, and multiple
+     * mappings exist, inserting a Contact record without populating any of the mapped fields does not
+     * generate any relationships records.
+     *********************************************************************************************************/
     @isTest
-    private static void insertContactWithMultipleAutoCreateMappingsNonePopulated(){
-
-        REL_Relationships_Con_AfterInsert_MTST.setupMultipleAutoCreateRelationships();
```
Comment (MatthewBlanski) [inline unpackaged/test_packaged/relationship_autocreate/classes/REL_Relationships_Con_PTST.cls:194]: The third actual code change

**Pull request merged**